README.md

```diff
@@ -28,3 +28,153 @@ Tentative roadmap:
 
 - VPN API
   - [ ] VPN API for desktop (Linux, Windows, macOS)
+
+# Cross-platform Development
+
+## Building
+
+In Go, you can compile for other target operating system and architecture by specifying the `GOOS` and `GOARCH` environment variables.
```

Comment:
I think it's worth mentioning that if you are using native integrations (such as `cgo`), this won't work.

Reply:
Added a comment about Cgo.